service/src/main/java/com/zebrunner/reporting/service/email/EmailType.java

```diff
@@ -1,12 +1,19 @@
 package com.zebrunner.reporting.service.email;
 
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
 public enum EmailType {
```

Comment:
Please remove this enum. You can't use enums literally everywhere. If those are used in 1 class - make static final vars in that class. If those are shared across many places create class with constants. Have you ever thought why media types or response codes are not enums in Spring framework?

Reply:
Then the same enum in admin-service should be removed as well

Reply:
I think I can. Provide more documented details to avoid my confusion